### ISSUE TITLE
Tahun pelajaran di client muter terus

### ISSUE THREAD

**Issue by merzakk** [eraporsmk/erapor7#20]
**Deskripsi yang jelas dan ringkas, sertakan screenshot/gif jika diperlukan**
Aplikasi yang sedang dipakai:
1. Operating System: 
2. Versi Aplikasi: 7.0.3
3. Versi Database: 6.00

Admin, untuk tahun pelajaran di client muter terus, mohon solusinya
![IMG_20231106_074908_385](https://github.com/eraporsmk/erapor7/assets/119062563/c0a2bcc4-75dd-46c5-b9bd-a6ba9b697b4b)

**Comment by masadi** [comment]
Pastikan APP_URL di file .env sudah sama dengan IP/Domain di address bar

**Comment by merzakk** [comment]
Jika di ubah IP di. Env, brrti untuk admin masuknya tidak lagi localhost pak?

**Comment by masadi** [comment]
Betul, akses URL nya harus konsisten, baik akses di server atau di client, URL nya harus sama.
Jika di akses berbeda, biasanya ada beberapa fitur gak jalan, seperti tahun ajaran, logo gak tampil dll

**Comment by merzakk** [comment]
Mohon maaf pak, kalau yg versi 7.02 kan, yang di server pakai localhost, sedangkan klient pakai IP bisa akses, tp kalau di 7.03 harus di buat IP di URLnya, ketika buka bersamaan antara server dan client, justru client malah masuk ke admin, bagaimana pak solusinya?

**Comment by masadi** [comment]
client masuk ke admin gimana ya pak?

**Comment by merzakk** [comment]
Ketika saya coba menggunakan user pengguna untuk entri, yang masuk langsung ke beranda admin pak... Apakah di admin harus di log out dahulu za

**Comment by masadi** [comment]
kalo mau login multi user, pake browser lain pak, atau pake incognito

**Comment by merzakk** [comment]
Terimakasih pak, saya cobanya

**Comment by muhammadhabibi54** [comment]
Permisi kasus saya sama Tahun pelajaran di client muter terus, sudah di coba ganti pakai IP di file env-nya hasilnya masih sama, mohon sulusinya

**Comment by WARDI-NURYANTO** [comment]
> Permisi kasus saya sama Tahun pelajaran di client muter terus, sudah di coba ganti pakai IP di file env-nya hasilnya masih sama, mohon sulusinya

sudah diupdate sama pengembang 3 jam yang lalu pak silahkan run admin file update-erapor.bat

**Comment by muhammadhabibi54** [comment]
Sudah ada update terbaru ya pak, baik pak saya cek dulu

**Comment by muhammadhabibi54** [comment]
Ok pak terima kasih sudah bisa pak

**Comment by HarisMI40** [comment]
Permisi saya juga mengalami kasus saya sama Tahun pelajaran di client muter terus, sudah di coba ganti pakai nama domain di file env-nya hasilnya masih sama,
ini pakai commit yang terbaru

**Comment by Ile02** [comment]
Saya juga mengalami hal yang sama. erapor saya online kan bisa di cek di erapor.smkn1soppeng.my.id
Jika dibuka menggunakan HTTP tidak ada kendala tetapi jika menggunakan HTTPS tahun pelajaran mutar-mutar terus.
Masalanya di browser otomatis mengarahkan koneksi ke HTTPS,
Sudah ubah file env tetapi masih belum teratasi
Mohon solusinya
